packages/devtools_app/lib/src/debugger/key_sets.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import '../config_specific/host_platform/host_platform.dart';
+
+final LogicalKeySet focusLibraryFilterKeySet = LogicalKeySet(
+  HostPlatform.instance.isMacOS
+      ? LogicalKeyboardKey.meta
+      : LogicalKeyboardKey.control,
+  LogicalKeyboardKey.keyP,
+);
+
+final LogicalKeySet goToLineNumberKeySet = LogicalKeySet(
+  HostPlatform.instance.isMacOS
+      ? LogicalKeyboardKey.meta
+      : LogicalKeyboardKey.control,
+  LogicalKeyboardKey.keyG,
+);
+
+final LogicalKeySet searchInFileKeySet = LogicalKeySet(
+  HostPlatform.instance.isMacOS
+      ? LogicalKeyboardKey.meta
+      : LogicalKeyboardKey.control,
+  LogicalKeyboardKey.keyF,
+);
+
+final LogicalKeySet escapeKeySet = LogicalKeySet(
+  LogicalKeyboardKey.escape,
+);
+
+final LogicalKeySet openFileKeySet = LogicalKeySet(
```

Comment:
add a TODO to change this to P once the library filter is removed.

Reply:
Done, thanks!